### ISSUE TITLE
equations rendered at 100%, look pixelated at 150%

### ISSUE THREAD

**Issue by ywwg** [pkgw/webtex#5]
I browse at 150% because I have a hi-DPI screen -- equations and graphs seem to be rendered assuming 96dpi

![webtext zoom](https://cloud.githubusercontent.com/assets/888940/6359003/b584affe-bc3d-11e4-8ecb-bb5aab839ef7.png)

**Comment by pkgw** [comment]
Hmmm, it's all done with <canvas> elements so it's not like I'm specifically hardcoding a DPI or anything. You're using Chrome, I assume?

**Comment by ywwg** [comment]
no, firefox, but I do see the same effect in Chrome.

Check out dom-render.js:~165.  It looks like a bunch of assumptions about point size to pixel ratios are made.  Just doing a 200% zoom in your browser should be enough to show you what's going on.

Boo, 8 space tab characters and 4 space indenting!

**Comment by pkgw** [comment]
Yes, I remember that code, I wrote it :-) I don't think it should affect the DPI with which canvases are rendered or anything, but I'll look into it. I'd certainly like to remove the hardcodey bits anyway.

I'm old-fashioned. How are the kids indenting these days?

**Comment by ywwg** [comment]
It depends on the style guide, but generally the use of tabs is discouraged.  Just use spaces.
